test/test.js

```diff
@@ -0,0 +1,97 @@
+const { expect } = require("chai");
```

Comment:
Even after creating my own hardhat config file, these tests don't run because `ethers` is not defined. After creating and checking in a hardhat config, please make sure the tests run here and and that things are properly imported, etc...

Reply:
It should be fixed now that the config file is included.

Reply:
Tests run and pass